docs/authentication.md

```diff
@@ -4,6 +4,8 @@ When using HTTPS, Helios client can present an X.509 client certificate to verif
 the user's identity. By presenting a digital signature to the Helios master, the
 client proves that it is who it claims to be.
 
+Note that Helios itself only contains functionality that lets the client present certificates. The Helios masters do not verify certificates themselves. The cluster administrator must add the missing pieces that validate certificates and act accordingly.
```

Comment:
We can add:

Only the Helios client supports HTTPS. The master doesn't. One needs a web server like nginx sitting in front to handle the TLS handshake and termination.

This web server can validate the client cert itself or by asking another service that checks a keystore like ldap. We could even open-source the x509-verifier and include sample nginx configs.